optuna/study/_study_direction.py

```diff
@@ -5,14 +5,11 @@ class StudyDirection(enum.IntEnum):
     """Direction of a :class:`~optuna.study.Study`.
 
     Attributes:
-        NOT_SET:
-            Direction has not been set.
         MINIMIZE:
             :class:`~optuna.study.Study` minimizes the objective function.
         MAXIMIZE:
             :class:`~optuna.study.Study` maximizes the objective function.
     """
 
-    NOT_SET = 0
```

Comment:
How about keeping `NOT_SET` since it exists until we provide the data migration for RDBStorages?

Reply:
I see.